.github/workflows/ci.yml

```diff
@@ -104,7 +104,25 @@ jobs:
           CI: true
       - run: bash <(curl -s https://codecov.io/bash)
   check-postgres:
-    name: Postgresql
+    strategy:
+      matrix:
+        include:
+          - name: Postgres 10, Postgis 3.1
+            POSTGRES_IMAGE: postgis/postgis:10-3.1
+            POSTGRES_MAJOR_VERSION: 10
+          - name: Postgres 11, Postgis 3.1
+            POSTGRES_IMAGE: postgis/postgis:11-3.1
+            POSTGRES_MAJOR_VERSION: 11
+          - name: Postgres 12, Postgis 3.1
```

Comment:
Why are there two 12 majors?

Reply:
This was to test against Postgis 3.0, and Postgis 3.1. I chose to test the Postgres 12 image twice

Reply:
We had a breaking change one point last year when Postgis made a change to some of their methods

Reply:
So what happens if image `postgis/postgis:12-3.2` is released, how will we find this out?

Reply:
> So what happens if image postgis/postgis:12-3.2 (= Postgres 3.2) is released, how will we find this out?

This is what I mentioned was similar to what mongo check. Currently it would need to be added manually. Postgis seems to maybe an update like that maybe once a year.

Reply:
Here are postgis versions, though part of their page seems to be broken right now https://postgis.net/news/

Reply:
Their dev docs show the minor version changes better: https://postgis.net/development/

Reply:
Got it, it would be nice if the CI could detect that a new minor is released and fail until we update the version in the CI, like it does for Mongo. But if Postgres only releases so few times a year that's not really an issue I guess.

Reply:
Just to support my claim about the Postgis minor releases:

- 3.1, Released Sometime recent, I can't find the date right now
- 3.0, Released 10/2019
- 2.5, Released 9/2018
- 2.4, Released, 9/2017
- 2.3, Released, 9/2016

Any updates in between his like 3.1.1...3.1.9 are automatically bumped to the latest because of the docker image.

Postgres updates are captured (these occur more often than Postgis updates) because the postgis image is using the major version of Posgres

Reply:
Sounds good!